sdk/digitaltwins/Azure.DigitalTwins.Core/tests/QueryTests.cs

```diff
@@ -6,13 +6,17 @@
 using System.Text.Json;
 using System.Threading;
 using System.Threading.Tasks;
+using Azure.Core.TestFramework;
 using FluentAssertions;
 using NUnit.Framework;
 
 namespace Azure.DigitalTwins.Core.Tests
 {
     public class QueryTests : E2eTestBase
     {
+        private const int RetryCount = 10;
+        private const int DelayInSeconds = 2;
```

Comment:
How about make this a TimeSpan?

Reply:
I am using a TimeSpan when using this variable in the call to RetryAsync. I can make this one TimeSpan instead.

Reply:
Saves you from having to create that type each time, and this can be static readonly

Reply:
And then also, the variable doesn't have to have the unit in there, just `s_delay`, or something slightly more descriptive like `s_retryDelay`